scalafmt-tests/src/test/resources/default/DefDef.stat

```diff
@@ -303,3 +303,19 @@ def foo =
     val bar = fetchBar()
     bar.map(_ + 10)
   })
+<<< #915 Space on both sides of operator method definition (positive)
+trait Test[A] {
```

Comment:
We should create a separate `test/afterSymbolicDefs.stat` file with these test cases

```
spaces.afterSymbolicDefs = true
<<< #915 Space on both sides of operator method definition (positive) 
...
>>>
...
````

It would be good to have a test case to validate that by default there is no space.